### PR TITLE
Fix formatting issues

### PR DESCRIPTION
**Pull request by sumedhe**
### Proposed changes in this pull request

Fix some code style issues

### CODE REVIEW

**Review by Kanapriya — APPROVED** [review]

File: components/claim-mgt/org.wso2.carbon.identity.claim.metadata.mgt/src/main/java/org/wso2/carbon/identity/claim/metadata/mgt/listener/ClaimMetadataTenantMgtListener.java [hunk 1]
```diff
@@ -1,3 +1,21 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (Kanapriya) [inline components/claim-mgt/org.wso2.carbon.identity.claim.metadata.mgt/src/main/java/org/wso2/carbon/identity/claim/metadata/mgt/listener/ClaimMetadataTenantMgtListener.java:2]: change the year in the license header to 2021

Comment (sumedhe) [reply components/claim-mgt/org.wso2.carbon.identity.claim.metadata.mgt/src/main/java/org/wso2/carbon/identity/claim/metadata/mgt/listener/ClaimMetadataTenantMgtListener.java]: Fixed

File: components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java [hunk 2]
```diff
@@ -5170,17 +5170,16 @@ private int createServiceProvider(String tenantDomain, ServiceProvider servicePr
     /**
      * Add audit log entry.
      *
-     * @param action Action
-     * @param data Audit data
-     * @param result Result
+     * @param action The action of the log.
+     * @param data Data of the action to log.
+     * @param result The success of fail state of the action.
```
Comment (somindatommy) [inline components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java:5175]: align the param descriptions

File: components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java [hunk 3]
```diff
@@ -3572,8 +3572,8 @@ public void deleteApplication(int applicationID, Connection connection)
     /**
      * Delete all applications of a given tenant id.
      *
-     * @param tenantId Id of the tenant
-     * @throws IdentityApplicationManagementException
+     * @param tenantId The id of the tenant
```
Comment (somindatommy) [inline components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java:3575]: Add a `.` at the end

File: components/consent-mgt/org.wso2.carbon.identity.consent.mgt/src/main/java/org/wso2/carbon/identity/consent/mgt/listener/TenantConsentMgtListener.java [hunk 4]
```diff
@@ -63,8 +63,8 @@ public void onPreDelete(int tenantId) throws StratosException {
     /**
      * Delete all consents belongs to a given tenant id.
      *
-     * @param tenantId Id of the tenant
-     * @throws StratosException
+     * @param tenantId The id of the tenant
```
Comment (somindatommy) [inline components/consent-mgt/org.wso2.carbon.identity.consent.mgt/src/main/java/org/wso2/carbon/identity/consent/mgt/listener/TenantConsentMgtListener.java:66]: Shall we add a `.` at the end?

**Review by somindatommy — APPROVED** [review]

File: components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java (hunk 2 above)
Comment (sumedhe) [reply components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java]: Fixed

File: components/consent-mgt/org.wso2.carbon.identity.consent.mgt/src/main/java/org/wso2/carbon/identity/consent/mgt/listener/TenantConsentMgtListener.java (hunk 4 above)
Comment (sumedhe) [reply components/consent-mgt/org.wso2.carbon.identity.consent.mgt/src/main/java/org/wso2/carbon/identity/consent/mgt/listener/TenantConsentMgtListener.java]: Fixed

File: components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java (hunk 3 above)
Comment (sumedhe) [reply components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java]: Fixed

File: components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/dao/WorkflowDAO.java [hunk 5]
```diff
@@ -266,6 +268,8 @@ public void removeWorkflowParams(int tenantId) throws InternalWorkflowException
                 prepStmt.setInt(1, tenantId);
                 prepStmt.executeUpdate();
                 IdentityDatabaseUtil.commitTransaction(connection);
+            } catch (SQLException e1) {
```
Comment (somindatommy) [inline components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/dao/WorkflowDAO.java:271]: can we use another variable here?

File: components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/dao/WorkflowDAO.java [hunk 6]
```diff
@@ -151,6 +151,8 @@ public void removeWorkflows(int tenantId) throws InternalWorkflowException {
...
```
Comment (somindatommy) [inline components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/dao/WorkflowDAO.java:154]: can we use another variable here?

File: components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/dao/WorkflowDAO.java (hunk 5 above)
Comment (sumedhe) [reply components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/dao/WorkflowDAO.java]: Fixed

File: components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/dao/WorkflowDAO.java (hunk 6 above)
Comment (sumedhe) [reply components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/dao/WorkflowDAO.java]: Fixed

**Review by somindatommy — APPROVED** [review]

**Review by somindatommy — APPROVED** [review]

**Pull request merged**